lib/assessment.md

```diff
@@ -138,7 +166,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+All of the divs will be oriented towards the left side of the page except divOne and divFour. divOne will orient itself to the right side of the page and divFour will clear all other elements and sit below the other elements.
```

Comment:
You're almost right. Even though div Four clears a left float, divs Four and Five are still floated, so div Five will sit immediately to the right of div Four.